templates/kubernetes/terraform/modules/kubernetes/backend_service.tf

```diff
@@ -0,0 +1,24 @@
+data "aws_secretsmanager_secret" "by-name" {
+  name = "<% .Name %>_cf_keypair"
```

Comment:
This should be `var.project` instead of the templated variable. As much as possible in the terraform we try to keep the templating in the environment main.tf file, and then pass those values as arguments to the modules.

Reply:
Ah, cool! Thanks!